build.gradle

```diff
@@ -138,17 +138,17 @@ dependencies {
     implementation 'io.prometheus:simpleclient_httpserver'
     implementation 'io.prometheus:simpleclient_log4j2'
 
-    implementation("com.squareup.okio:okio:3.8.0") {
+    implementation("com.squareup.okio:okio:3.4.0") {
```

Comment:
thought: we should leave a note about why we downgraded the library versions and we should update them once we migrate to kotlin 1.9